speech.d.ts

```diff
@@ -21,6 +21,60 @@ declare module 'lmnt-node' {
 
     /** The desired target length of the output speech in seconds. */
     length?: number;
+
+    /** If `True`, the response will include word durations detail. */
+    return_durations?: boolean;
+
+    /** If `True`, the response will include the seed used for synthesis. */
+    return_seed?: boolean;
+  }
+
+  /**
+   * Options available to pass with streaming synthesis requests. See the
+   * [SDK documentation](https://lmnt.com/docs/node/) for available values
+   * and defaults.
+   */
+  interface StreamingSynthesisOptions {
+    /** Whether to return extra data (durations data and warnings) with each audio chunk. */
+    return_extras?: boolean;
+
+    /** The desired speed of the synthesized speech. */
+    speed?: number;
+
+    expressive?: number;
+  }
+
+  interface FetchVoicesOptions {
+    /** If true, only returns voices that you have starred. Defaults to false. */
+    starred?: boolean;
+
+    /** Specify which voices to return. Choose from `lmnt`, `me`, or `all`. Defaults to `all`. */
+    owner?: 'lmnt' | 'me' | 'all';
+  }
+
+  interface UpdateVoiceOptions {
+    /** The name of the voice */
+    name?: string;
+
+    /** Whether the voice is starred */
+    starred?: boolean;
+
+    /** The gender of the voice, e.g. `male`, `female`, `nonbinary`. For categorization purposes. */
+    gender?: string;
+    
+    /** A description of the voice */
+    description?: string;
+  }
+
+  interface CreateVoiceOptions {
+    /** The type of voice to create. Must be one of `instant` or `professional`. Defaults to `instant`. */
+    type?: string;
+
+    /** The gender of the voice, e.g. `male`, `female`, `nonbinary`. For categorization purposes. Defaults to `null`. */
+    gender?: string;
```

Comment:
Maybe I'd say "undefined" or "unset" or "unknown". Do we reveal "null" or display it that way in a UI? Same for below re: description ("empty" for that, maybe, by default).

Reply:
good question, I think there's nothing displayed in the UI at the moment so totally should change the default. In the REST api, these fields are actually none

Reply:
I lied, the default in rest for gender is U